nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/sqs/DeleteSQS.java

```diff
@@ -45,7 +46,8 @@
 public class DeleteSQS extends AbstractSQSProcessor {
 
     public static final PropertyDescriptor RECEIPT_HANDLE = new PropertyDescriptor.Builder()
-            .name("Receipt Handle")
+            .name("sqs-receipt-handle")
```

Comment:
Thanks for reviewing, @brosander.

Ordinarily, I would agree that changing the name is bad, but I don't believe that is the case here.  The `RECEIPT_HANDLE` property descriptor is not actually added to the current list of property descriptors on the processor.  The diffs for lines 57-60 don't make it crystal clear, but the problem is that the processor uses this property descriptor without ever actually declaring it for the user to configure.  It works by coasting on the default value expression.  Since it has never been added to a DeleteSQS processor in an as-is version of NiFi, I don't believe we have the upgrade problem you expect.

I'm certainly not attached to changing the name, I just thought it would be a good practice under the circumstances.  I would be happy to take that out if you aren't convinced.